app/_hub/kong-inc/response-transformer/index.md

```diff
@@ -69,44 +65,56 @@ params:
     - name: remove.headers
       required: false
       value_in_examples: ["x-toremove", "x-another-one"]
+      datatype: array of string elements
       description: List of header names. Unset the header(s) with the given name.
     - name: remove.json
       required: false
       value_in_examples: ["json-key-toremove", "another-json-key"]
+      datatype: array of string elements
       description: List of property names. Remove the property from the JSON body if it is present.
     - name: rename.headers
       required: false
-      description: List of original_header_name:new_header_name pairs. If the header `original_headername` is already set, rename it to `new_headername`. Ignored if the header is not already set.
+      datatype: array of string elements
+      description: List of `original_header_name:new_header_name` pairs. If the header `original_headername` is already set, rename it to `new_headername`. Ignored if the header is not already set.
     - name: replace.headers
       required: false
-      description: List of headername:value pairs. If and only if the header is already set, replace its old value with the new one. Ignored if the header is not already set.
+      datatype: array of string elements
+      description: List of `headername:value` pairs. If and only if the header is already set, replace its old value with the new one. Ignored if the header is not already set.
```

Comment:
do we have some way of representing the `one_of` schema field here in the docs?

Reply:
> do we have some way of representing the `one_of` schema field here in the docs? https://github.com/Kong/kong-ee/blob/master/kong/plugins/response-transformer/schema.lua#L66

@murillopaula do you have any suggestions? We don't want the type field to be overloaded/crowded. We can clarify in the description.

Reply:
yeah clarifying in the description seems good to me. This plugin is already doing it: https://github.com/Kong/docs.konghq.com/pull/2617/#discussion_r580996833

Reply:
I just realized i've been inconsistent with _Array of string elements_ (sentence case) vs _array of string elements_.